### PR TITLE
Contact-related UI/UX enhacements

### PR DESCRIPTION
**Pull request by rock-ejin**
Fixes #support [67](https://github.com/CTemplar/support/issues/67)
### Changes description

  - Allow to "save and add another contact"
  - Favorite/Starred contacts.

### CODE REVIEW

**Comment by The-Hidden-Hand** [comment]
@rock-ejin 
Do we need the console.log?

File: src/app/mail/mail-contact/mail-contact.component.ts [hunk 1]
```diff
@@ -184,6 +201,19 @@ export class MailContactComponent implements OnInit, AfterViewInit {
...
+  toggleStarred(contact: Contact) {
+    contact.starred = !contact.starred;
+    if (contact.is_encrypted) {
+      this.openpgp.encryptContact(contact);
+    } else {
+      this.store.dispatch(new ContactAdd(contact));
+    }
+  }
+
+  onSave(): any {
```
Comment (superdev0714) [inline src/app/mail/mail-contact/mail-contact.component.ts:213]: What does this function stand for?

File: src/app/mail/mail-sidebar/compose-mail/receiver-email-chip/receiver-email-chip.component.ts [hunk 2]
```diff
@@ -106,6 +106,9 @@ export class ReceiverEmailChipComponent {
       },
     });
   }
+  onSaveContact() {
+    console.log('Contact saved');
```
Comment (superdev0714) [inline src/app/mail/mail-sidebar/compose-mail/receiver-email-chip/receiver-email-chip.component.ts:110]: Remove this console.

**Review by superdev0714 — CHANGES_REQUESTED** [review]
Please check the comments

**Comment by rock-ejin** [comment]
I removed consoles and empty functions

**Pull request merged**